docs/How-to-test-a-PR.md

```diff
@@ -60,6 +60,14 @@ The commands above will create a new local branch called `pr-XXXX` which will co
 
 The terminal might prompt an editor (usually `nano`) that asks to save the merge commit message. Just save the changes and exit the editor. If the editor is `nano`, you can do it by simply using `CTRL+O` and `ENTER` to save then `CTRL+X` to exit.
 
+This may also be the way to resolve the editor merge commit message prompt.
+
+    press "i" (i for insert)
+    write your merge message
+    press "esc" (escape)
+    write ":wq" (write & quit)
+    then press enter
+
```

Comment:
good catch, it looks like on Windows the default editor is `vim` and not `nano` as mentioned in [this article](http://web.mit.edu/6.005/www/fa14/tutorial/git/config.html), but I found a better way of saving and existing.

Can you please check if this works for you?

Reply:
@FrancescoBorzi @JamesGent i already added info about the default editor in this PR, not sure what we should do (not gonna duplicate the paragraphs) https://www.azerothcore.org/wiki/How-to-create-a-PR

maybe we should have a side page that we link to? not sure rn and i don't want to think too much sorry xD